auslib/AUS.py

```diff
@@ -35,23 +46,18 @@ def getFallbackChannel(channel):
 class AUSRandom:
     """Abstract getting a randint to make it easier to test the range of
```

Comment:
The new version of this class is merely a tiny wrapper around randint() now - can we just call it directly, instead? I don't know that enforcing the min/max values really matters all that much since the caller is just a few lines down.

Reply:
I thought it was nice to have a clear "statement" that the min/max values the random number generator should return are 0 and 99. But I've gotten rid of it.